deployment/docker-compose/README.md

```diff
@@ -20,33 +23,33 @@ The TDX device in the guest has different names in certain versions
 To enable the containers read and write the TDX device, change the access privilege of the TDX device.
 
 ```
-chmod 0666 $(find /dev/ -name "tdx*")
+$ chmod 0666 $(find /dev/ -name "tdx*")
 ```
 
 This tool will generate files to create some docker composes which are place in `/tmp/docker_ccnp`. Please make sure the directory clear.
 
 For convenience, run the script `prepare.sh` directly as root.
 
 ```
-sudo ./prepare.sh
+$ sudo ./prepare.sh
 ```
 
-## 2. Use CCNP
+## Deploy CCNP
 
-### 2.1 Deploy CCNP
+### Deploy CCNP Services
 
-Use the script `deploy-ccnp.sh` to deploy the CCNP services. 
+Use the script [deploy-ccnp.sh](./depoly-ccnp.sh) to deploy the CCNP services. 
 
 ```
-./deploy-ccnp.sh
+$ ./deploy-ccnp.sh
 ```
 
 In default, the script will launch three containerized services
```

Comment:
It seems like there should be a colon or a period here.

Reply:
Updated. Thanks.